Week06/src/MainInventory.java

```diff
@@ -0,0 +1,152 @@
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+import java.util.Scanner;
+
+public class MainInventory {
+
+    public static void main(String[] args) {
+
+        Scanner sc = new Scanner(System.in);
+
+        System.out.println("Welcome! Choose the number from the options below to continue:");
+        System.out.println("1. Create product and add it to stock");
+        System.out.println("2. Sell product");
+        System.out.println("3. Display daily sales and report");
+        System.out.println("4. Exit");
+
+        List<Product> ProductList = new ArrayList<>();
+        List<Order> OrderList = new ArrayList<>();
+        
+        ProductList.add(new Animal("Pate", 11, 30, 1, 5, "2020.10.10", 4));
+        ProductList.add(new Animal("Milk", 12, 10, 1, 20, "2019.09.20", 4));
+        ProductList.add(new Animal("Sausage", 13, 20, 3, 50, "2019.12.25", -20));
+        ProductList.add(new Animal("Cheese", 14, 5, 3, 12, "2019.9.30", 4));
+        ProductList.add(new Animal("Dog", 15, 200, 2, 40, "2050.01.01", 40));
```

Comment:
There was a constraint that the animal products are made of milk or ar milk and eggs... I would have liked to see a check that the products match the requirement somewhere. You are allowing any animal product to be sold